user_guide/data_management.rst

```diff
@@ -0,0 +1,234 @@
+need to know its unique id (number), how many items are in the cart, what their total value is, whether a purchase has 
+been made. Therefore, we have an entity “Cart”, records of which have all these properties: number of items, total 
+value, status. 
+
+One customer may have several carts. For OroCRM this means that Web Customer records are related to Carts 
+records as one to many. To show this, we assign one of the Web Customer properties as a “relation” to carts. This 
+field will save identificators of the customer’s carts (such identificator is a field of the cart). By the 
+identificator the system can find the cart and bind its properties to the customer. 
+
+Moreover, each cart may have several 
+items in it. This means that each Cart record is related to its Item records as one to many. 
+
+*User-wise, this means that you can get easily see the full picture and drill-down into details as much as required for 
+any specific task, as well as update the process the provided data.*  
+
+See All the Records of a Kind on Grids
+--------------------------------------
+
+Grid is an aggregated view of all the records of the same entity. Each row of the grid is one records, and each column 
+is one of the grid properties. 
+
+For example, you can open a grid of all the Contacts. (The Contact records represent actual people you are dealing with 
+and their properties are personal information, details of their position in the customer-company, contact details, etc.
+The contacts may be related to a customer, an opportunity, or any other record in the system, if reasonable.)
+
+So, we have opened the *"All Contacts"* grid. 
+
+.. image:: ./img/data_management/grid/grid_01.png
+
+We immediately see some major details of our contacts, such as their first and last names, email, phone number, etc.
+
+You can adjust and save grid views, updated and delete grid records, get to the Edit and View pages of any grid record, 
+and even export all the data in the grid as a .csv file.
+
+The grid details are described in the :ref:`Grids <user-guide-ui-components-grids>` guide.
+
+View Details of a Specific Record
+---------------------------------
+
+To see more details of a specific record, and to work directly with the record (create a task related to a customer, 
+appoint a calendar event for a user, turn a cart into an order, share a contact and so on), you need to get to its 
+View page.
+
+For example, we have opened the View page of one of our Contacts.
+
+.. image:: ./img/data_management/view/view_01.png
+
+On the View page you can see all the details of the record and location of the address on the map. You can 
+click the phone number link to call it via Hangouts or log a call, start writing an email with a click on the address, 
+initiate Skype session directly from the View page. If there is a related entity, its identifier will be a link to the 
+View page of this entity
```

Comment:
Edit: On the View page, you can see all the details of a record and the location of its address on the map. You can click the phone number link to call it via Hangouts or log a call, start writing an email by clicking on its address, or initiate a Skype session directly from the View page. If there is a related entity, its identifier will be a link to the View page of this entity.